packages/bcer-data-portal/app/src/views/Locations.tsx

```diff
@@ -513,11 +561,13 @@ export default function Locations() {
                   noi_report: searchTerms.noi_report ? searchTerms.noi_report: 'all',
                   product_report: searchTerms.product_report ? searchTerms.product_report: 'all',
                   manufacturing_report: searchTerms.manufacturing_report ? searchTerms.manufacturing_report: 'all',
-                  sales_report: searchTerms.sales_report ? searchTerms.sales_report: 'all'
+                  sales_report:  searchTerms.sales_report ? searchTerms.sales_report: 'all',
+                  fromdate: searchTerms.fromdate ? searchTerms.fromdate : 'all',

```

Comment:
You shouldn't use 'all' here, confirm from mui docs what empty value is for calender. Should be "" or null

Reply:
should be null, updated